### PR TITLE
chor(download-links): Add 3.9.1 download links

### PR DESCRIPTION
**Pull request by andromaqui**

### CODE REVIEW

File: enterprise/content/download.md [hunk 1]
```diff
@@ -2577,13 +2577,13 @@ To get the production distribution, please download one of the following archive
...
+      <td>3.9.1</td>
       <td>7.16+, 7.17+, 7.18+</td>
       <td>7.13.0+, 7.14.0+, 7.15.0+, 7.16.2+, 7.17.0+</td>
-      <td>11.10.2022</td>
+      <td>13.10.2022</td>
       <td>
-        <a class="btn btn-sm btn-default optimize-download" href="https://downloads.camunda.cloud/enterprise-release/optimize/3.9.0/camunda-optimize-3.9.0-production.zip">zip</a>&nbsp;
-        <a class="btn btn-sm btn-default optimize-download" href="https://downloads.camunda.cloud/enterprise-release/optimize/3.9.0/camunda-optimize-3.9.0-production.tar.gz">tar.gz</a>
+        <a class="btn btn-sm btn-default optimize-download" href="https://downloads.camunda.cloud/enterprise-release/optimize/3.9.1/camunda-optimize-3.9.1-production.zip">zip</a>&nbsp;
+        <a class="btn btn-sm btn-default optimize-download" href="https://downloads.camunda.cloud/enterprise-release/optimize/3.9.1/camunda-optimize-3.9.1-production.tar.gz">tar.gz</a>
```
Comment (grlimacan) [inline enterprise/content/download.md:2586]: I know this was not from you, but the preview2 should be the entry immediately below 3.9.1, so you need to swap it with preview 1

File: enterprise/content/download.md [hunk 2]
```diff
@@ -2577,13 +2577,13 @@ To get the production distribution, please download one of the following archive
   <tbody>
     <tr class="well">
       <td><a href="/optimize">Camunda Optimize</a></td>
-      <td>3.9.0</td>
```
Comment (grlimacan) [inline enterprise/content/download.md:2580]: Github doesnt let me comment on unchanged lines, so I'm writing this here: When doing a minor release, you need to remove the alpha releases from the table with the list of alpha releases starting on line 2724

**Review by grlimacan — CHANGES_REQUESTED** [review]
Some corrections required

File: enterprise/content/download.md (hunk 2 above)
Comment (andromaqui) [reply enterprise/content/download.md:2580]: oh right! I will add this to our docs then :)

**Review by grlimacan — APPROVED** [review]
LGTM!

**Pull request merged**